posthog/async_migrations/migrations/0006_persons_and_groups_on_events_backfill.py

```diff
@@ -75,9 +76,19 @@ class Migration(AsyncMigrationDefinition):
             int,
         ),
         "GROUPS_DICT_CACHE_SIZE": (1000000, "ClickHouse cache size (in rows) for groups data.", int),
+        "TIMESTAMP_LOWER_BOUND": ("2020-01-01", "Timestamp lower bound for events to backfill", str),
+        "TIMESTAMP_UPPER_BOUND": ("2022-10-01", "Timestamp upper bound for events to backfill", str),
+        "TEAM_ID": (
+            None,
+            "The team_id of team to run backfill for. If unset the backfill will run for all teams.",
+            str,
+        ),
     }
 
     def precheck(self):
+        # Used to guard against self-hosted users running on `latest` while we make tweaks to the migration
+        if not settings.TEST and get_instance_setting("ALLOW_EXPERIMENTAL_ASYNC_MIGRATIONS"):
+            return (False, "ALLOW_EXPERIMENTAL_ASYNC_MIGRATIONS is set to False")
```

Comment:
Is this intended? `ALLOW_EXPERIMENTAL_ASYNC_MIGRATIONS` would be true if this is reached

Reply:
ah oops I reused the `not` when I popped in `settings.TEST` - will update

good catch!